osgpkitools/incommon_request.py

```diff
@@ -70,6 +70,7 @@ def parse_cli():
     '''%(prog)s [--debug] -u username -k pkey -c cert \\
            (-H hostname | -F hostfile) [-a altnames] [-d write_directory]
        %(prog)s [--debug] -u username -k pkey -c cert -t
+       %(prog)s [--orgcode org,dept] (-H hostname | -F hostfile) -u username -k pkey -c cert
```

Comment:
Same comments apply here as to README.md

Reply:
Fixed